### DISCUSSION TITLE
NTSC - QT Edition Preset Merge to NTSC RS

### DISCUSSION THREAD

**Discussion by Nikos2Evil** [ntsc-rs/ntsc-rs#169] [General]
Hello, is there anyway I can take a preset I have made in QT & import it to RS. If so how can I do it? Tt states that they are unsupported.

**Comment by Degamisu** [comment]
The NTSC-RS and QT preset jsons im pretty sure have different formatting, and different settings. thus, im afraid it wont be possible. I could probably help you recreate them though!

**Reply by Nikos2Evil** [reply]
I would appreciate that, I will drop the .txt file with the preset below so that maybe you can figure out how to recreate it in RS.
[qt preset.txt](https://github.com/user-attachments/files/16742025/qt.preset.txt)

**Reply by Degamisu** [reply]
> I would appreciate that, I will drop the .txt file with the preset below so that maybe you can figure out how to recreate it in RS. [qt preset.txt](https://github.com/user-attachments/files/16742025/qt.preset.txt)

Alright. I'll see what i can do. This should take me around 30 mins to an hour.

**Reply by Degamisu** [reply]
That actually took a lot less time than I expected!

Here are some demo pics:

<details><summary>NTSC-QT (Original)</summary>
<p>

![NTSCQT-TEST](https://github.com/user-attachments/assets/b1985af8-4c3b-4819-bcfd-55f68dd5e6a1)

</p>
</details> 

<details><summary>NTSC-RS</summary>
<p>

![image](https://github.com/user-attachments/assets/8f97c2ec-45e6-47cf-bcbd-efef404e8d6f)

</p>
</details> 

Anyways, Here's the preset. If you want to tweak it, go for it! Or want me to change something, let me know!

[Nikos2Evil-QT.json](https://github.com/user-attachments/files/16748095/Nikos2Evil-QT.json)

**Reply by Nikos2Evil** [reply]
I appreciate it very much, this worked out well. Thank you.

**Reply by Degamisu** [reply]
> I appreciate it very much, this worked out well. Thank you.

No problem! Anytime.
